src/modules/blogs/blogs.service.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-prototype-builtins */
```

Comment:
Why disable the linter here?

Reply:
I didnt work on that feature and I want keep the changes I make minimal but I have resolved it now.
The linter was throwing an error on that service